src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -70,6 +71,19 @@ public static AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    /**
+     * Returns an {@code AddressBook} with all the typical persons in randomized order.
+     */
+    public static AddressBook getTypicalReversedAddressBook() {
```

Comment:
If its randomized, why is it called reversed?

Reply:
The comment was not updated to the newest state yet